Symfony/CS/Tests/Fixer/PSR2/BracesFixerTest.php

```diff
@@ -32,6 +32,20 @@ public function provideFixControlContinuationBracesCases()
         return array(
             array(
                 '<?php
+class Foo
+{
+    public function A()
+    {
+        ?>
+        Test<?php echo $foobar; ?>Test
+        <?php
+        $a = 1;
```

Comment:
the problem is that adding new tokens to collection is slow, and doing it in transformer means doing it for all code.